### PR TITLE
[PD-922] Made templates UTF-8 by default and changed Configuration Table to UTF-8

### PR DESCRIPTION
**Pull request by aspidites**
Changed the Configuration table in seo to use UTF-8 and added a chareset line to each of the base templates.

### CODE REVIEW

**Comment by galitskyd** [comment]
:+1: @JLMcLaugh

File: templates/base.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
     {% block site-title %}
     <title>My.jobs</title>
     <meta name="title" content="My.jobs">
+    <meta charset= “utf-8”>
```
Comment (tdphillips) [inline templates/base.html]: Fancy quotes intentional?

Comment (aspidites) [inline templates/base.html]: In so much as it was a copy-paste, yes. Otherwise no. I wonder if i fix that if i can get rid of the unit test changes...

**Comment by aspidites** [comment]
Much better. Thanks @tdphillips

**Pull request merged**